resources/assets/coffee/react/beatmap-discussions/header.coffee

```diff
@@ -121,6 +121,11 @@ class BeatmapDiscussions.Header extends React.PureComponent
               beatmap: @props.currentBeatmap
 
 
+  discussionUsers: =>
+    userIds = _.uniq(_.map(_.values(@props.discussions), 'user_id'))
+    _.pickBy(@props.users, (value) -> _.includes(userIds, value.id))
```

Comment:
Here have a magic thingy chained written in javascript.

    _($r.props.discussions).map('user_id').uniq().map(function(k){return $r.props.users[k]}).sortBy('username').value()